### PR TITLE
Publishing August 5 2016 Meeting Notes

### PR DESCRIPTION
**Pull request by TheLarkInn**
@webpack/collab-team 
@webpack/contributor-team 

Meeting notes are up! Let me know if I need to add or change anything.

### CODE REVIEW

File: 2016-08/august-05.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+* Currently the `webpack/webpack.io` is a statically generated site, so Juho is going to be working to add React and JS for Dynamic usage in other portions of the page. 
+
+* Documentation Structure has been finalized!!!! Priority number one is now creating the content and merging it into develop! The styling for the page is still a work in progress, however that is our last priority before we "ship the docs". 
+
+* For future we may have Johannes with the help of other UI Designers to create personas that help us and contributors create appropriate language for documentation. Agreed that it was a very-nice-to-have but also a great addition if we ever had time. 
+
+* Would love to have folks work on a ShemaStore schema for the webpack configuration. This we could use to autogenerate some documentation like the API: Configuration page. Also could provide IDE support for people almost out of the box. Need to look into how to do this when we tackle that documentation.
```
Comment (bebraw) [inline 2016-08/august-05.md]: /s/ShemaStore/SchemaStore

File: 2016-08/august-05.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+* For future we may have Johannes with the help of other UI Designers to create personas that help us and contributors create appropriate language for documentation. Agreed that it was a very-nice-to-have but also a great addition if we ever had time. 
+
+* Would love to have folks work on a ShemaStore schema for the webpack configuration. This we could use to autogenerate some documentation like the API: Configuration page. Also could provide IDE support for people almost out of the box. Need to look into how to do this when we tackle that documentation. 
+
+* It was brought up that we need to create a feasible action plan that we can share with the Dan Abramov's, Kent Dodd's, and Patrick Stapleton's of the JS community to pass along and promote. It is in everyones best interest to work together to create documentation and having a strategy to accomplish this is vital. We spent time listing out the immediate MVP documents that would the team agreed could allow us to life the webpack 2.x beta status.
```
Comment (bebraw) [inline 2016-08/august-05.md]: /s/Dodd's/Dodds'

Comment (TheLarkInn) [inline 2016-08/august-05.md]: Nice catch Kent will appreciate it.

File: 2016-08/august-05.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+## July 29 ([discuss](https://github.com/webpack/meeting-notes/pull/8))
```
Comment (bnjmnt4n) [inline 2016-08/august-05.md:1]: This should be August 5 ;)

File: 2016-08/august-05.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+* For future we may have Johannes with the help of other UI Designers to create personas that help us and contributors create appropriate language for documentation. Agreed that it was a very-nice-to-have but also a great addition if we ever had time. 
+
+* Would love to have folks work on a SchemaStore schema for the webpack configuration. This we could use to autogenerate some documentation like the API: Configuration page. Also could provide IDE support for people almost out of the box. Need to look into how to do this when we tackle that documentation. 
+
+* It was brought up that we need to create a feasible action plan that we can share with the Dan Abramov's, Kent Dodds', and Patrick Stapleton's of the JS community to pass along and promote. It is in everyones best interest to work together to create documentation and having a strategy to accomplish this is vital. We spent time listing out the immediate MVP documents that would the team agreed could allow us to life the webpack 2.x beta status.
```
Comment (kentcdodds) [inline 2016-08/august-05.md:29]: 👋  😄 
I'm not 100% certain I understand what this bullet point is trying to say. Happy to promote Webpack stuff :)

File: 2016-08/august-05.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+* Juho created a [Github Milestone for all these tasks that we can give to people](https://github.com/webpack/webpack.io/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Webpack+2+-+Documentation+MVP%22)!!!! Barring any major or breaking changes from popular community loaders or plugins (as well as the core loaders and plugins), once this is finished, we believe this is our 'go-ahead' to lift the beta status. We all agreed that although this wouldn't contain all of the documentation we aspired to tackle, that this would give the (first time) users _that need it the most_, the opportunity to have a _far more_ augmented learning experience than what is available. 
+
+* Everyone also agreed that we could technically accomplish this by the end of August. **Yes people, it's getting more than real, this is an estimated timeline of webpack 2.x being out of beta!** :heart_eyes: :heart_eyes: :heart_eyes:. If all goes well and we meet our deadline, then we can lay out our next milestone for more advanced "revisions" of our existing documentation migrated over to the `webpack/webpack.io` repo.
```
Comment (kentcdodds) [inline 2016-08/august-05.md:33]: 😍 😍 😍 😍

File: 2016-08/august-05.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+* **We will want/need lots of help!!!** So please share the link of the milestone above. The milestone will also include general guidelines for how to author one of these docs. And once we have a collection of people we can meet weekly via hangout, etc. to track and manage the Milestone goals, etc. There is also a [getting started page](https://github.com/webpack/webpack.io/blob/develop/content/get-started.md) we have referenced on the repo as well for those worried about contributing. It doesn't have to be perfect the first time around. 
+
+#### Publication
+
+* Sean admitted he has slacked a little bit on getting a new article for our publication, but wants to make it a priority to announce that the angular-cli and react-create-app are both powered by webpack now. 
+
+* Its a huge milestone of integration for us. Including Laravel and Rails:Middleman and Grails new asset Pipline. The list of webpack users is seeming to continue to increase and the learning and constructive criticism continues to grow. This _is the tool you made and helped customize_ in the end, so every person, framework, library, boilerplate, MVC, that uses webpack is another win for you and the webpack team.
```
Comment (kentcdodds) [inline 2016-08/august-05.md:41]: As a long-time webpack user who kind of bet the farm on webpack, seeing the widespread adoption has been really exciting!

File: 2016-08/august-05.md (hunk 4 above)
Comment (TheLarkInn) [inline 2016-08/august-05.md:29]: You are 3 of many people who a have strong representative for JavaScript and Webpack. Nothing more than a compliment.👏

Comment (kentcdodds) [inline 2016-08/august-05.md:29]: I took it as a compliment. Thanks! It just looks like you might be asking us to do something. Wanna know what you want me to do to help :)

File: 2016-08/august-05.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+#### Attendee Notes:
+Tobias is going to be on vacation for a couple weeks and thus wasn't present for this meeting.
+
+### Agenda
+
+#### Documentation Updates
+
+* Sean has [merged](https://github.com/webpack/webpack.io/pull/59) our first [new documentation](https://github.com/webpack/webpack.io/blob/develop/content/concepts/index.md) into webpack/webpack.io!!!! Share it around, and provide feedback. It is still in a beta state as there is some link cleanups. But he wanted to provide a great introduction to people who haven't already had one. 
+
+* We are still on the hunt for the `webpack.io` domain name. We see its registered to a gentleman in China, and would love any Mandarin speakers to help us draft an email to reach out to him for domain purchase as the first few emails have been unsuccessful!! [Marco Pai](https://github.com/MarcoPai) has some slight activity on github. So we aren't giving up hope just yet.
```
Comment (e-cloud) [inline 2016-08/august-05.md:19]: if it doesn't make any progress, I may give a hand on it.

File: 2016-08/august-05.md (hunk 5 above)
Comment (lcxfs1991) [inline 2016-08/august-05.md:33]: I am willing to help.
My name is Lee, working in Tencent.
You can contact me via lcxfs1991@gmail.com or add my [facebook](https://www.facebook.com/lee.li.311)

Comment (lcxfs1991) [inline 2016-08/august-05.md:33]: I have written a post to help you locate this guy.
[http://gold.xitu.io/post/57a69cd8a633bd00604ade55](http://gold.xitu.io/post/57a69cd8a633bd00604ade55)

**Pull request merged**